routing/router.go

```diff
@@ -612,6 +612,10 @@ type BlindedPathRestrictions struct {
 	// NodeOmissionSet is a set of nodes that should not be used within any
 	// of the blinded paths that we generate.
 	NodeOmissionSet fn.Set[route.Vertex]
+
+	// IncomingChainedChannels holds a route of chained channels that we
+	// should use as incoming hops during blinded path selection.
```

Comment:
Nit: specify the order of the channels in that list, counting from the destination backwards ....